cross/x86_64-binutils/Portfile

```diff
@@ -24,10 +24,14 @@ if {$subport eq $name} {
 foreach ostarget {linux dragonfly freebsd netbsd openbsd} {
     subport x86_64-${ostarget}-binutils {
         crossbinutils.setup     x86_64-${ostarget} ${version}
+        depends_lib             port:$name
```

Comment:
I'm confused. Does this make the port depend on itself!?

Reply:
No. Only the subports x86_64-${ostarget}-binutils depend on x86_64-binutils, which installs the docs once for all subports. Seems to work.

Reply:
Put in a comment maybe about this so people reading in the future aren't confused the way I was.